cartridges/int_algolia/cartridge/scripts/algolia/steps/algoliaProductDeltaIndex.js

```diff
@@ -328,7 +334,13 @@ exports.afterStep = function(success, parameters, stepExecution) {
     // Any prior return statements will set success to false (even if it returns Status.OK).
 
     if (!jobReport.error) {
-        if (jobReport.chunksFailed > 0) {
+        const failurePercentage = +((jobReport.recordsFailed / jobReport.recordsToSend * 100).toFixed(2)) || 0;
```

Comment:
This might not behave as expected: `toFixed()` converts the number into a string.

Reply:
The `+` in front of it converts it back to a number: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Unary_plus

Reply:
So then why not use `round()`, `trunc()` or `floor()`?****